site/zenodo_rdm/assets/semantic-ui/less/zenodo_rdm/previewer/mirador-previewer.less

```diff
@@ -39,4 +46,8 @@ i.info.icon {
     content: "i";
     color: #F8FFFF; 
     font-weight: 600;
+  }
+
+  h3, p {
+    font-family: 'Lato','Helvetica', Arial, sans-serif;
```

Comment:
I would scope these to the parent CSS class container so that the style does not leak.